package/rmt-server.changes

```diff
@@ -4,6 +4,7 @@ Tue Oct 30 15:35:17 UTC 2018 - skotov@suse.com
 - Version 1.0.7
 - Import migration_extra attribute from SCC API
 - Auto-selected modules feature migration engine changes
+- Restore old behavior for ASMM in online migrations
```

Comment:
I don't think we need this line. Maybe the line before can be worded more customer friendly: 
`Offline migrations will automatically add additional modules to the client systems depending on the base product`

Reply:
fixed